tests/unit-tests/enrolment/test-class-sensei-course-manual-enrolment-provider.php

```diff
@@ -103,28 +103,10 @@ public function testIsEnrolledLegacyNoMigrateNoProgress() {
 		$this->assertNotEmpty( get_option( 'sensei_enrolment_legacy' ), 'The site-wide legacy enrolment flag should have been set.' );
 
 		$is_enrolled = $provider->is_enrolled( $student_id, $course_id );
-		$this->assertTrue( $this->wasLegacyEnrolmentChecked( $student_id, $course_id ), 'Legacy enrolment status for user should have been checked.' );
+		$this->assertFalse( $this->wasLegacyEnrolmentChecked( $student_id, $course_id ), 'Legacy enrolment status for user should not have been checked.' );
 		$this->assertFalse( $is_enrolled, 'The user should not have been enrolled because they were not enrolled previously.' );
 	}
 
-	/**
-	 * Tests to make sure enrolment is provided when the `sensei_is_legacy_enrolled` filter returns true.
-	 */
-	public function testIsEnrolledLegacyMigrateWithFilter() {
```

Comment:
We could keep this test and legacy enrol, but it wouldn't be testing anything really.